lib/nine_one_one/slack_service.rb

```diff
@@ -18,10 +21,15 @@ def notify(message)
 
     private
 
-    attr_reader :http, :path
+    attr_reader :channel, :http, :path, :username
 
     def request_body(message)
-      { text: message }.to_json
+      body = { text: message }
+
+      body[:channel]  = channel  unless channel.nil?
+      body[:username] = username unless username.nil?
```

Comment:
are nils even valid options here? that said, even if they ain't valid, the checks might still make sense - I'm just checking if you did add it consciously :)